src/Contracts/DocumentsQuery.php

```diff
@@ -31,11 +32,24 @@ public function setFields(array $fields): DocumentsQuery
         return $this;
     }
 
+    public function setFilter(array $filter): DocumentsQuery
```

Comment:
should add phpdoc what is `$filter` :)

Reply:
I don't. At least for new methods could be good to add :)

Reply:
I've added the phpdoc. Can you check it?

Reply:
Looking at the tests it's only list<string> or list<non-empty-string>

Or are there any cases with nested arrays?

Reply:
They are valid, I just don't need to test all the cases across every SDK (we are trying to avoid that as much as possible).

You can have this: `setFilter([['id > 10'], ['price < 250']])` which will translate to: "id > 10 AND price < 250"

Reply:
Okay so `list<non-empty-string|list<non-empty-string>>`

Reply:
done! ✅